pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -46,7 +46,7 @@ type BrokerList struct {
 
 // BrokerSpec represents a description of a Broker.
 type BrokerSpec struct {
-	// The URL to communicate with the Broker via..
+	// The URL to communicate with the Broker via.
```

Comment:
awkward phrasing if not a nonsense sentence fragment.

plus, not godoc. (Or are you allowed to have leading articles now?)

// URL is the address used to communicate with the broker.